weblab/entities/views.py

```diff
@@ -51,9 +53,10 @@
     FileUploadForm,
     ModelEntityForm,
     ProtocolEntityForm,
-)
+    EntityTransferForm)
```

Comment:
As a code style thing, it's best practice to keep the closing bracket on a line by itself, and add the extra import on its own line with a trailing comma, in alphabetical order.